### PR TITLE
Added F-Droid nightly

### PR DESCRIPTION
**Pull request by gouri-panda**

### CODE REVIEW

File: .github/workflows/fdroid_nightly.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Publish nightly build
+
+on:
+  pull_request:
+    branches:
+      - testing
```
Comment (gouri-panda) [inline .github/workflows/fdroid_nightly.yml:6]: I'll letter add the midnight. This is for manually testing nightly.

**Comment by kelson42** [comment]
This is still a draft? Is that full tested?

**Comment by gouri-panda** [comment]
@kelson42 It's not in the draft.

**Comment by kelson42** [comment]
So why, there is not a proper nighlty scheduling?

**Comment by gouri-panda** [comment]
>So why, there is not a proper nighlty scheduling?

The nightly won't trigger unless it merges in the develop.

**Review by kelson42 — APPROVED** [review]

**Pull request merged**